src/main/java/redis/clients/jedis/JedisClientConfig.java

```diff
@@ -80,4 +80,18 @@ default HostAndPortMapper getHostAndPortMapper() {
     return null;
   }
 
+  public interface ClientSetInfoConfig {
+
+    default boolean disable() {
+      return false;
+    }
+
+    default String libNameSuffix() {
```

Comment:
Please document here the requirements for the libNameSuffix:
`" " < c && c != "(" && c != ")" && c < "~"`